dev/atc.dev.sh

```diff
@@ -162,6 +162,7 @@ function tm-health-client {
 	return $?;
 }
 
+export API_VERSION=4.0
```

Comment:
Why bother making this modifiable if the only thing it affects is an endpoint that doesn't change with API version?

Reply:
Made `API_VERSION` unmodifiable in 741a09fba1

Reply:
Reverted 741a09fba1 in 93d477c1ca because if it is unmodifiable, sourcing `dev/atc.dev.sh` twice makes it print an error:
```bash
-bash: declare: API_VERSION: readonly variable
```

Reply:
I sort of meant why bother making it a variable at all. Just because it's undocumented that this value is exported to the execution environment, and somebody could possibly already be using it for something else.

Reply:
#6588 no longer uses `API_VERSION`, since it no longer defines a `to-req` function as of 57513153c0. IMO a `TO_API_VERSION` environment variable should still be defined somewhere so we don't hard-code the API version in URLs anywhere, but since this PR has no use for it, removed `API_VERSION` in c80f64501e.